docs/studio/cli/team.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Team
+description: Manage and inspect information about your Studio teams.
+keywords:
+  - CLI
+  - command line
+  - studio
+---
+
+## `studio team <subcommand>`
+
+Use these commands to interact with your Studio teams, including listing teams associated with your accounts, creating new teams, and adding members to teams.
+
+### `ls [identifier]`
+
+List all of your teams and underlying projects tied to your Studio account, optionally, passing the team ID.
+
+| Argument       | Type     | Default | Description                         |
+| -------------- | -------- | ------- | ----------------------------------- |
+| `[identifier]` | `string` | --      | (Optional) the team's ID, if known. |
+
+### `create <name> [personalTeamId]`
```

Comment:
not sure if the `personalTeamId` is correct / how it's used, but it was shown in the flags when i do `studio team create -h`.

but, i wasnt able to get this command to work—it shows the following and then never creates a team:
```
studio team create new_team_name
trying to create team...
doing create by pId...
```